am6uno/Algorithm-Run-Time-Complexity
# Teacher problem page

## Documentation
- Created a page for teachers to manage the problems in a set.
- Teachers can add, delete and edit problems.
- Additionally, teachers can see basic statistics about the solution submissions.

## Testing
- Restart your backend and docker containers.
- Go to the teachers problem page by selecting the temporary link in the nav menu.
- Verify that you can see the problems in the set. The link takes you to set 1. 
- Verify that you can delete problems.
- Verify that you can add problems to the set.
- Verify that you can edit problems.
- Create a solution either with the ui or postman to a problem and verify that the statistics show up.
